### ISSUE TITLE
[Spanish] localize 'Virtual Machine'

### ISSUE THREAD

**Issue by electrocucaracha** [cncf/glossary#1784]
Localize the `content/en/virtual-machine.md` document to Spanish

**Comment by Dheeraj12121** [comment]
what we have to do it here create new file ?

**Comment by electrocucaracha** [comment]
Yes, the idea is to create a new file under `content/es/` folder. There are many [similar PRs](https://github.com/cncf/glossary/pulls?q=is%3Apr+label%3Alang%2Fes+is%3Aclosed) that you can take a look on how and also [review the contribution process](https://glossary.cncf.io/es/contribute/). Usually, we recommend contributing to native speaking languages.

**Comment by marianogg9** [comment]
@electrocucaracha estoy disponible para trabajar en este issue. Gracias!

**Comment by ramrodo** [comment]
done by https://github.com/cncf/glossary/pull/1980
